ledger-service/http-json/src/main/scala/com/digitalasset/http/HealthService.scala

```diff
@@ -11,7 +11,7 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Try
 import scala.util.control.NonFatal
 
-private class HealthService(
+class HealthService(
```

Comment:
Making it public means some type hygiene is in order that doesn't matter for private classes; we should remove `extends StrictLogging`.

Reply:
```suggestion
final class HealthService(
```

Reply:
Remove `StrictLogging`